### PR TITLE
fix: use tab id instead of title for tabAffinity

### PR DESCRIPTION
**Pull request by andyschwob**
@bweissbart @wSedlacek 
- change type `Tab` to be a tab object only (not a string)
- set tabAffinity to be the id of the tab instead of the title

### CODE REVIEW

File: packages/fsapp/src/beta-app/router/history/utils.base.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ const matchPath = (path: string | undefined, route: Route) => {
 
 const buildMatcher = async (
   route: Route,
-  tab: Tab = '',
+  tab: Tab = { id: '' },
```
Comment (wSedlacek) [inline packages/fsapp/src/beta-app/router/history/utils.base.ts:69]: This should probably just be nullable with no default and use optional chaining on line 85.

A `tabAffinity` of `''` doesn't seem particularly useful or semantic.

**Review by wSedlacek — APPROVED** [review]
The changes look good, I did have one idea after reviewing this that you can play around with if you'd like.

**Comment by andyschwob** [comment]
@wSedlacek fair enough, i was just matching the old behavior.  It's updated now.

**Review by wSedlacek — APPROVED** [review]
LGTM

**Pull request merged**